trident/spectrum_generator.py

```diff
@@ -176,7 +184,8 @@ class SpectrumGenerator(AbsorptionSpectrum):
     def __init__(self, instrument=None, lambda_min=None, lambda_max=None,
                  n_lambda=None, dlambda=None, lsf_kernel=None,
                  line_database='lines.txt', ionization_table=None):
-        if instrument is None and lambda_min is None:
+        if instrument is None and (lambda_min is None or
```

Comment:
I realize that this modification makes sense based on what was originally here, but how come lambda_max isn't check in this scenario at all? @brittonsmith?

Reply:
No good reason I would say. I'll add it.